Description: avoid crate quickcheck
 Uses old release of quickcheck unavailable in Debian.
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2022-04-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -27,8 +27,8 @@
 
 [dev-dependencies]
 indexmap = "1.0"
-quickcheck = "0.6"
-rand = "0.4"
+#quickcheck = "0.6"
+#rand = "0.4"
 seahash = "3.0.5"
 serde = "1.0"
 serde_json = "1.0"
--- a/tests/header_map_fuzz.rs
+++ b/tests/header_map_fuzz.rs
@@ -1,3 +1,4 @@
+/*
 extern crate http;
 extern crate rand;
 extern crate quickcheck;
@@ -363,3 +364,4 @@
 
     String::from_utf8(bytes).unwrap()
 }
+*/
